### PR TITLE
[Polymer][SP] Release camera and restore state between active and blur.

### PR DESCRIPTION
**Pull request by DonnaWuDongxia**

### CODE REVIEW

**Comment by DonnaWuDongxia** [comment]
@leonhsl please take a look, thanks!

File: sample/src_polymer/elements/scene-perception-pages/main.js [hunk 1]
```diff
@@ -5,38 +5,43 @@ const SP_SIZE_FPS = 60;
...
+    // 'checking'(1) - init successfully,
+    // 'tracking'(2) - SP started
+    spDom.spState = 0;
   }
+  myStatus.info('Please wait for initialization.');
+  initSP(function() {
+    if (!spDom._initialized) {
+      leftView.init();
+      rightView.init();
+      spDom._initialized = true;
```
Comment (leonhsl) [inline sample/src_polymer/elements/scene-perception-pages/main.js]: No any code resetting `_initialized` to `false`? And, as we already have `spDom.spState`, maybe `_initialized` is redundant flag?

**Comment by DonnaWuDongxia** [comment]
@leonhsl updated, please help to review.

**Comment by leonhsl** [comment]
lgtm

**Comment by leonhsl** [comment]
Merged

**Pull request merged**